src/Compilers/Core/Portable/InternalUtilities/UICultureUtilities.cs

```diff
@@ -11,127 +11,23 @@ namespace Roslyn.Utilities
 {
     internal static class UICultureUtilities
     {
-        // TODO (DevDiv 1117307): Replace with CultureInfo.CurrentUICulture.set when available.
```

Comment:
This seems like an overall fine change but need some reasoning why it's in this PR.

Reply:
This code is entirely useless now. CultureInfo.CurrentUICulture.set is available in netstandard2.0.

Reply:
Right but why put it in this PR vs. just send a PR that fixes this. Not sure I understand the fit in this PR.

Reply:
Oh, it uses reflection. I have to either delete it or annotate.